src/main/java/rest/client/RestClient.java

```diff
@@ -0,0 +1,55 @@
+package rest.client;
```

Comment:
rename packages as rest_client

Reply:
Could you please point me to the docs about this?
I have found another view on that in Google codestyle 
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names